sql/extra/extract_patient_profile_photo_from_db.sql

```diff
@@ -0,0 +1,24 @@
+DROP PROCEDURE IF EXISTS dump_image;
+DELIMITER //
+  CREATE PROCEDURE dump_image()
+  BEGIN
+
+	DECLARE done INT;
+    DECLARE this_id INT;
+    DECLARE cur1 CURSOR FOR SELECT PAT_PROFILE_PHOTO_ID FROM OH_PATIENT WHERE PAT_PROFILE_PHOTO_ID IS NOT NULL;
```

Comment:
Why are the `DECLARE` statements lined up on the same column?

Reply:
Ok, got it and fixed. Thanks!